docs/SEMAPHORE_GUIDE.md

```
@@ -0,0 +1,352 @@
+
+```bash
+# Clone the repository
+git clone https://github.com/linuxmalaysia/podman-elastic-stack-ai.git
+
+# Navigate into the project directory
+cd podman-elastic-stack-ai
+```
+
+For more details on cloning and initial repository setups, see the [Git Repository guide in INSTALL.md](INSTALL.md#git-repository).
+
+### Enable User Linger
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)